internal/font.go

```diff
@@ -15,6 +15,7 @@ type Font interface {
 	GetStyle() consts.Style
 	GetSize() float64
 	GetFont() (consts.Family, consts.Style, float64)
+	GetFontSize() (ptSize, unitSize float64)
```

Comment:
You have GetSize and GetFontSize inside Font interface, both return float 
I suggest to add some comments to interface and maybe divide this interface, or rename one of the methods.

Reply:
I removed this, now the interface has a GetScalarFactor() and I apply this factor to the response from GetFont(). This Scalar Factor is a low level detail inside gofpdf.